todo.py

```diff
@@ -0,0 +1,16 @@
+from datetime import date
+from typing import Optional
+from dataclasses import dataclass
+
+# DTO - Data Transfer Object
+
+@dataclass
+class Todo:
+    name: str
+    due_date: date
+    description: Optional[str] = None
+    is_done: bool = False
+    # user_id: int
```

Comment:
Acho que pode excluir essa parte aqui, meio nada a ver ela. No mais ficou xou @rafaeltedesco 
```suggestion
    # user: int
```